src/core/containers/ServerHtml.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom/server';
+import serialize from 'serialize-javascript';
+import Helmet from 'react-helmet';
+
+
+export default class ServerHtml extends Component {
+
+  static propTypes = {
+    appName: PropTypes.string.isRequired,
+    assets: PropTypes.object.isRequired,
+    component: PropTypes.object.isRequired,
+    env: PropTypes.string.isRequired,
+    sriData: PropTypes.object,
+    store: PropTypes.object.isRequired,
+  };
+
+  getStatic({filePath, type} = {}) {
```

Comment:
This is the first I've seen something like this. I guess it allows calling `getStatic()` to set `filePath` and `type` to `undefined` though?